src/libgrate/fragment_asm_parser.y

```diff
@@ -765,6 +767,19 @@ TEX_INSTRUCTION:
 	|
 	;
 
+TEX_LOD_REG:
+	',' T_ROW_REGISTER
+	{
+		asm_tex_instructions[asm_fs_instructions_nb].enable_bias = 1;
+		yyval.u = $2;
+	}
+	|
+	{
+		asm_tex_instructions[asm_fs_instructions_nb].enable_bias = 0;
+		yyval.u = 0xDEAD;
```

Comment:
Ah, okay :) I just quickly glanced through the TGSI ISA without paying much attention. So the TEX opcode is fine as it is now, no need to change anything about it.